### ISSUE TITLE
Remove information concretizations (SDCs)

### ISSUE THREAD

**Issue by wdduncan** [OBOFoundry/COB#108]
Remove specifically dependent concretizations that concretized information.

**Comment by cmungall** [comment]
Related #107 - do in the same PR?

**Comment by jamesaoverton** [comment]
I don't think we had consensus on this at the COB workshop. For my part, in general, I'm opposed to removing concretization.

**Comment by cmungall** [comment]
Onus is on those who would prefer to see this remove to make a concrete proposal: @cmungall

**Comment by jamesaoverton** [comment]
Closing old issue.

**Comment by wdduncan** [comment]
I understand this issue is old, but I would like to keep it open since it hasn't been settled.

Re [comment](https://github.com/OBOFoundry/COB/issues/108#issuecomment-685906477):    
> Onus is on those who would prefer to see this remove to make a concrete proposal: @cmungall 

The proposal is simply to remove the axiom. E.g. plan as has the axiom "concretizes some plan specification".  What else do you need?

My reasoning for this is published [here](https://jbiomedsem.biomedcentral.com/articles/10.1186/s13326-024-00315-0).

**Comment by jamesaoverton** [comment]
The last line of the abstract is "However, more research is needed to test this hypothesis as well as explore the other solutions presented in this manuscript." Where is the concrete proposal for COB?

**Comment by wdduncan** [comment]
@jamesaoverton The term `plan` needs to have the axiom removed.  

Please leave the issue open.
